Sources/MapboxCoreNavigation/EHorizon/RoadObjectMatcherDelegate.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import MapboxNavigationNative
+
+/**
+ `RoadObjectMatcher` delegate.
+ */
+public protocol RoadObjectMatcherDelegate: class {
```

Comment:
Probably makes sense to update to:
```suggestion
public protocol RoadObjectMatcherDelegate: AnyObject {
```

to prevent build warnings in Xcode `12.5`.